README.md

```diff
@@ -1,6 +1,59 @@
-# lit-element
+# LitElement
 
-## Base class for creating custom elements using Polymer and lit-html.
+## A simple base class for creating custom elements rendered with lit-html.
+
+LitElement uses [lit-html](https://github.com/Polymer/lit-html) to render into the
+element's Shadow DOM and [Polymer's](https://github.com/Polymer/polymer)
+PropertiesMixin to help manage element properties and attributes.
+
+LitElement uses a functional reactive pattern where any changes to element
+properties are batched and then rendered asynchronously. Users describe
+the entire rendering state declaratively, and lit-html ensures that updates
+are fast by creating the static DOM once and smartly updating only the parts of
+the DOM that change.
+
+For basic usage:
+
+  1. Create a class that extends `LitElement`.
+  1. Implement a static `properties` getter that returns the element's properties
+  (which automatically become observed attributes).
+  1. Then implement a `_render(props)` method and use the element's
```

Comment:
So is this the only required method to implement for a class that extends `LitElement`?

Reply:
yah!

Reply:
Done